content/news/015.open-tech-meetup-3.de.md

```diff
@@ -0,0 +1,17 @@
+---
+published: 2025-05-26
+description: Neue Software, Hardware und Projekte werden der Dresdener Netzwerk-Community vorgestellt
+title: DD-IX Open Tech Meetup 2025-05
+keywords: [ Event ]
+authors: [ Maurice ]
+image: 015_open_tech_meetup_3.jpg
+---
+
+Das OTM-2025-05 (Open Tech Meetup) fand erneut in den Räumlichkeiten der TU Dresden statt.
+Das DD-IX Team präsentierte die neuen Peers, speziell BCIX Outreach und dessen Funktionalität, die Software IXpect (Developmentname audix) um das Peering LAN auf ungewollte Pakete zu auditieren und das Lab, welches benutzt wird um die neu erhaltene Hardware vom [DD-IX Opening](https://dd-ix.net/en/news/dd-ix-opening) möglichst gut auf neue Features zu testen.
```

Comment:
Das DD-IX Team präsentierte die neuen Peers, speziell BCIX Outreach, die vom DD-IX entwickelte Open-Source-Software IXpect, um ungewollte Pakete im Peering LAN zu erkennen, 
und das DD-IX Lab, in dem wir die neu erhaltene Hardware vom [DD-IX Opening](https://dd-ix.net/en/news/dd-ix-opening) auf vielversprechende Features vorab testen.